install.sh

```diff
@@ -45,6 +45,8 @@ arch=$(uname -sm)
 case "${arch}" in
     "Linux x86_64") download $name-$version-x86_64-unknown-linux-musl ;;
     "Linux i686") download $name-$version-i686-unknown-linux-musl ;;
+    "Linux aarch64") download $name-$version-aarch64-unknown-linux-gnu ;;
+    "Linux armv7"*) download $name-$version-armv7-unknown-musleabihf ;;
```

Comment:
Yeah, from Googling it, it looks like `uname -m` reports `armv7l` and maybe `armv7`, `armv7hf`,  `armv7h`, and `armv7m` . It looks like it has to do with what type of floating-point precision the kernel is reporting from the CPU. Since rust only targets one armv7 arch with two libc ABIs (musl and glibc), I am pretty sure this will work, but I could change it to be `armv7l` knowing that it will work with the possibility of missing other armv7 architechtures...

It was really tricky to piece together what `uname -m` would report vs what ARM lists on their wiki etc.. https://en.wikipedia.org/wiki/Uname says `armv7l` for sure...

Reply:
Also, as far as I understand it, `rustc` automatically statically links musl libcl for triples with musl but creates a dynamically linked one for glibc (gnu, triples)... is that correct? i.e. there will not be a run-time dependence on musl libc which is why you had x86 use the musl triple?

Reply:
OK, just keep this `arv7*` for now until someone complains.

Yes, the difference is runtime dependencies on libc or not. Please update this to use musl.

Reply:
I wanted to use musl but it looks like `cross` doesn't support it yet for `aarch64`... rust-embedded/cross#168

Reply:
If I have time I can submit a PR to cross to add support for it...

Reply:
Ahh, you're right. I will merge this in.

Reply:
Good luck for the CR to cross!

Really wish rust cross compiling would become better.

Reply:
Ok, thanks! I will submit a PR to update it to `musl` once cross adds support for it.

Also, just so you are aware, there are some OSes that will report `Linux aarch64` and `Linux armv7l` but do not have the same ABI that the `aarch64-unknown-linux-gnu` and `armv7-unknown-linux-musleabihf` triplets require. Android is one that I know of for sure (it requires `aarch64-linux-android`), but there are probably others - which is why I updated INSTALL.md to note that...

Reply:
I thought about adding a check based on `uname -o` for `GNU/Linux` but I figured there could potentially be exotic distros that report something else for `uname -o` that run on the Linux kernel... Also, it ruins the nice one-line switch statement :)

Reply:
Good to know these. Thanks.